testapi.pm

```diff
@@ -1353,14 +1301,13 @@ Special characters naming:
 
 =cut
 
-sub send_key {
-    my $key = shift;
+sub send_key ($key, @) {
     my %args = (@_ == 1) ? (do_wait => +shift()) : @_;
```

Comment:
`@_` and `shift`

Reply:
yes, I see. What do you suggest?

Reply:
The [documented signature](https://open.qa/api/testapi/#_send_key) is `send_key($key [, wait_screen_change => $wait_screen_change]);` but I guess we need this for backward compatibility... maybe just leave it, since it'll look awkward even if you give the array argument a name

Reply:
for completeness, I don't see the problem with replacing `@_` with `@args`:
```
sub send_key ($key, @args) {
    my %args = (@args == 1) ? (do_wait => +shift args) : @args;
```
I think this was mentioned elsewhere or in a nother signature PR already.
But you decided to leave these for now. Just mentioning my solution.

Reply:
ok, left it in the old way for now.